.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+#        go: [1.8, 1.9, '1.10']
```

Comment:
Trial and error leftovers?

Reply:
Was going to left it for the next PR, but fixed now.